messaging/kafka-sarama/base.go

```diff
@@ -106,12 +111,34 @@ func New(option *Option) (messaging.QueueV2, error) {
 
 func (l *Kafka) NewListener(option *Option) (*cluster.Consumer, error) {
 	config := cluster.NewConfig()
+
 	config.Consumer.Return.Errors = true
-	config.Consumer.MaxWaitTime = l.Option.MaxWait
+	// config.Consumer.MaxWaitTime = l.Option.MaxWait
 	config.Group.Return.Notifications = true
-	config.Group.PartitionStrategy = l.Option.Strategy
-	config.Group.Heartbeat.Interval = time.Duration(l.Option.Heartbeat) * time.Second
+	// config.Group.PartitionStrategy = l.Option.Strategy
```

Comment:
Can you mention why we are removing these ?

Reply:
I commented them for testing and forgot to uncomment 🤦